dex/farm/src/lib.rs

```diff
@@ -339,6 +343,9 @@ pub trait Farm:
     ) -> SCResult<CompoundRewardsResultType<Self::Api>> {
         require!(self.is_active(), "Not active");
 
+        let caller = self.blockchain().get_caller();
```

Comment:
Compound does not apply to this type of farm contract. This could be removed if we are aiming for a smaller code base.

Reply:
agreed. good call.